app/models/user.rb

```diff
@@ -1,7 +1,8 @@
 class User < ApplicationRecord
+    has_secure_password
+    
     validates :first_name, presence: true
     validates :last_name, presence: true
     validates :email, presence: true, uniqueness: true
-    validates :password, presence: true
-    validates :password_confirmation, presence: true
+    validates :password_digest, presence: true
```

Comment:
We don't need to validate that the password_digest is present. The has_secure_password is all we need